app.py

```diff
@@ -38,8 +38,77 @@ def mirror(name):
     }
     return create_response(data)
 
+
+
 # TODO: Implement the rest of the API here!
 
+
+@app.route('/users', methods = ['GET'])
+def users():
+    if ('team' in request.args):
+        team = request.args.get('team')
+        data = db.get('users')
+        ret = []
+        for item in data:
+            if (item['team'] == team):
+                ret.append(item)
+
+        return create_response(ret)
+    else:
+        data = db.get('users')
+        return create_response(data)
+
+@app.route('/users/<userID>', methods = ['GET'])
+def usersID(userID):
+    data = db.get('users')
+    if (int(userID) > len(data) or int(userID) <= 0):
+        return create_response({}, 404, 'No such user exists!')
```

Comment:
Maybe you could define `STATUS_NOT_FOUND = 404` as a global in this module and use that here. Same with the other status codes.